.github/workflows/compile.yml

```
@@ -0,0 +1,23 @@
+      image: texlive/texlive:latest  # Или texlive/texlive:small (ещё меньше)
+    steps:
+      - uses: actions/checkout@v4
+      - run: |
+          # Установка biber (если его нет в базовом образе)
+          apt-get update && apt-get install -y biber
+          
+          # Сборка LaTeX
+          pdflatex -synctex=1 -interaction=nonstopmode -file-line-error main.tex
+          biber main
+          pdflatex -synctex=1 -interaction=nonstopmode -file-line-error main.tex
+          pdflatex -synctex=1 -interaction=nonstopmode -file-line-error main.tex
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)